src/odin/maneuversbuilder.cc

```diff
@@ -2277,5 +2324,46 @@ void ManeuversBuilder::EnhanceSignlessInterchnages(std::list<Maneuver>& maneuver
   }
 }
 
+void ManeuversBuilder::ProcessTurnLanes(std::list<Maneuver>& maneuvers) {
```

Comment:
walks the maneuvers and activates the proper turn lanes at each transition point